src/assets.ts

```diff
@@ -356,19 +385,18 @@ function getBuildOperations(tasksJsonPath: string) {
                     }
 
                     const text = buffer.toString();
-
-                    let buildTask: tasks.TaskDescription;
+                    let tasksConfiguration: tasks.TaskConfiguration;
 
                     try {
-                        const tasksJson: tasks.TaskConfiguration = tolerantParse(text);
-                        buildTask = tasksJson.tasks.find(td => td.taskName === 'build');
+                        tasksConfiguration = tolerantParse(text);
                     }
                     catch (error) {
                         vscode.window.showErrorMessage(`Failed to parse tasks.json file`);
-                        buildTask = undefined;
```

Comment:
If we catch an exception here, we will still call getBuildTasks. Is that what we want?